cmd/go-clang-generate/function.go

```diff
@@ -135,3 +139,50 @@ func generateFunctionEqual(f *Function) string {
 
 	return b.String()
 }
+
+var templateGenerateStructMemberGetter = template.Must(template.New("go-clang-generate-function-getter").Parse(`{{$.Comment}}
+func ({{$.Receiver}} {{$.ReceiverType}}) {{$.Name}}() {{if $.IsReturnTypePointer}}*{{end}}{{if $.IsReturnTypeArray}}[]{{end}}{{if $.ReturnPrimitiveType}}{{$.ReturnPrimitiveType}}{{else}}{{$.ReturnType}}{{end}} {
```

Comment:
that would probably be a good idea
however, i would suggest to do this at a later point (e.g., final refactoring)